gamestonk_terminal/fred/README.md

```diff
@@ -0,0 +1,46 @@
+# fred
+
+The purpose of fred is to have the ability to look at economic data.
+
+* [GDP](#GDP)
```

Comment:
Would be nice to have this command in lower case to match the remaining style of the program.